arch/arm/soc/st_stm32/stm32f0/dts.fixup

```diff
@@ -85,4 +85,46 @@
 #define CONFIG_PWM_STM32_17_DEV_NAME            ST_STM32_PWM_40014800_PWM_LABEL
 #define CONFIG_PWM_STM32_17_PRESCALER           ST_STM32_PWM_40014800_PWM_ST_PRESCALER
 
+#ifdef ST_STM32_USART_40013800_RS485_ENABLED_AT_BOOT_TIME
+	#define CONFIG_UART_STM32_USART_1_RS485_ENABLED			1
```

Comment:
Might want to use some naming other than `CONFIG_...`, to make it clear that these aren't coming from Kconfig.

That might help with finding references to undefined Kconfig symbols later too.

Reply:
Seems most items in dts.fixup for stm32 series has a CONFIG_,.,, prefix, thats was aren't coming from Kconfig both. I prefer open another PR to remove the CONFIG_ prefix, not is this PR.

What's yours opinion?

Reply:
Fine by me, if you'd need to change a bunch of unrelated stuff.

Reply:
can you write an issue?

Reply:
Hmm... not sure how I'd phrase it. Might be other places with non-Kconfig `CONFIG_*` macros as well.

If we add some script for detecting references to undefined Kconfig symbols in source files later, all of them would probably have to be fixed at that point at least.